content/docs/user-guide/components/reference/gradients/image-gradient.md

```diff
@@ -26,38 +26,61 @@ configuration options, such as tiling, scaling, and custom sampling types.
-| **Mip Index** | Specify which mip level index to sample from.<br><br>If you specify a mip level higher than the number of mip levels available in your image, then the highest existing mip index will be used (see [`MipCountMax`](https://github.com/o3de/o3de/blob/development/Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/Limits.h)). | Int: `0` to `15` | `0` |
-| **Sampling Type** | The sampling type to be used on the image data. | `Point`, `Bilinear` | `Point` |
+| **Save Mode** | Specify how to choose the path for saving the image after any image edits. | `Save As...`, `Auto Save`, `Auto Save With Incrementing Names` | `Auto Save` |
 
 ### Sampling types
 
-There are several sampling types to choose from. To illustrate, here are previews of the same source image using the different supported sampling types.
-
-#### Point
+There are several sampling types to choose from. Choosing the best sampling type for a specific need is generally a balance between performance and quality.
 
-`Point` sampling is the default and only samples a single point for each pixel of the source image.
+| Sampling Type | Description | Performance Cost | Quality | Example |
+| - | - | - | - | - |
+| Point | `Point` sampling is the default and only samples a single point for each pixel of the source image.<br><br>This will produce blocky artifacts if the image is sampled at a higher frequency than the number of pixels. `Point` sampling produces the best results when the pixel size exactly matches the sampling frequency. | 1x | Poor | ![Image Gradient using point sampling](/images/user-guide/components/reference/gradients/image-gradient-component-point.png) |
+|  Bilinear | The `Bilinear` filter smooths out the image by requesting four points in a grid around a requested pixel and then performing interpolation between the points.<br><br>Because this uses linear interpolation between 4 points, bilinear filtering can cause noticeable plus-shaped artifacts in the smoothed data. `Bilinear` sampling is the best general-purpose choice, as it balances performance and quality. | 4x | Good | ![Image Gradient using bilinear interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bilinear.png) |
+|  Bicubic | The `Bicubic` filter smooths out the image by requesting sixteen points in a grid around a requested pixel and then performing Catmull-Rom interpolation between the points.<br><br>Because this uses non-linear interpolation, there are no noticeable plus-shaped artifacts in the smoothed data. `Bicubic` sampling is the best choice when quality is needed over performance. | 16x | Great | ![Image Gradient using bicubic interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bicubic.png) |
```

Comment:
```suggestion
|  `Bicubic` | The `Bicubic` filter smooths out the image by requesting sixteen points in a grid around a requested pixel and then performing Catmull-Rom interpolation between the points.<br><br>Because this uses non-linear interpolation, there are no noticeable plus-shaped artifacts in the smoothed data. `Bicubic` sampling is the best choice when quality is needed over performance. | 16x | Great | ![Image Gradient using bicubic interpolation](/images/user-guide/components/reference/gradients/image-gradient-component-bicubic.png) |
```

Reply:
This change should also be applied to Point and Bilinear?

Reply:
Yes, odd, I thought I made those suggestions.